File: js/romanNumerals.js
exports.toRoman = function(num) {
    var numbers = [1000, 900, 500, 400, 100, 90, 50, 40, 10, 9, 5, 4, 1];
    var symbols = ["M","CM","D","CD","C","XC","L","XL","X","IX","V","IV","I"];
    var output=''
    for(var i=0;i<13;i++){
      var quotient = Math.floor(num/numbers[i]);
      num=num%numbers[i];
      for(var j=0;j<quotient;j++){
        output+=symbols[i];
      }
    }  
    return output; 
};

File: python/roman_numerals.py
import math
def to_roman(num):
    # write your code here!
    numbers = [1000, 900, 500, 400, 100, 90, 50, 40, 10, 9, 5, 4, 1]
    symbols = ["M","CM","D","CD","C","XC","L","XL","X","IX","V","IV","I"]
    output=''
    for x in range(13):
      quo = math.floor(num/numbers[x])
      num=num%numbers[x]
      for y in range(quo):
        output+=symbols[x]
    return output